src/app/rpmostree-builtin-status.c

```diff
@@ -62,6 +63,7 @@ static GOptionEntry option_entries[] = {
   { "json", 0, 0, G_OPTION_ARG_NONE, &opt_json, "Output JSON", NULL },
   { "jsonpath", 'J', 0, G_OPTION_ARG_STRING, &opt_jsonpath, "Filter JSONPath expression", "EXPRESSION" },
   { "booted", 'b', 0, G_OPTION_ARG_NONE, &opt_only_booted, "Only print the booted deployment", NULL },
+  { "pending-exit-77", 'b', 0, G_OPTION_ARG_NONE, &opt_pending_exit_77, "If pending deployment available, exit 77", NULL },
```

Comment:
This will trigger if e.g. auto-staged updates are enabled, even if the user didn't install something.  Which...is fine, it's documented that way, but I do wonder if someone would want an option to detect whether a particular txn actually did anything or not.